packages/dmn-js-decision-table/src/features/decision-table-head/index.js

```diff
@@ -1,6 +1,7 @@
 import DecisionTableHeadProvider from './DecisionTableHeadProvider';
 
 export default {
+  __depends__: [ 'translate' ],
```

Comment:
Does this do anything? I think we must depend on a module. Depending on a `string` does not yield any meaningful result.